docs/design.md

```diff
@@ -30,6 +30,15 @@ parameters:
     type: int
     default: 3306
 ```
+Or with an empty parameters field
+```yaml
+name: fusor/etherpad-apb
+description: Note taking web application
+bindable: true
+async: optional
+parameters: []
```

Comment:
I should change this one to one that has no parameters. do we have an APB that would work for? Right now RDS - MySql is set up that way. Is there a better example that I should choose?